app/src/main/res/values/strings.xml

```diff
@@ -217,4 +213,40 @@
      When the described button is pressed, the overlay will open and the browser will be hidden underneath it -->
     <string name="hint_press_menu_to_open_overlay_a11y">Press the menu button for %1$s Home, search, settings, and more</string>
 
+    <!-- FxA Strings-->
+    <!-- Content description for a button that will prompt a user to sign in, in order to receive tabs pushed from FF
+    on another device -->
+    <string name="fxa_login_a11y">Receive tabs</string>
+    <!-- Heading for settings menu item button and CTA link on new update -->
+    <string name="fxa_settings_item_new">New! Firefox tabs on Fire TV</string>
+    <!-- Heading for settings menu item button and CTA link when signed in -->
+    <string name="fxa_settings_item_signed_in">Sent tab settings</string>
+    <!-- Heading for settings menu item button and CTA link when need to sign in again -->
+    <string name="fxa_settings_item_sign_in_again">Sent tabs - sign in again</string>
+    <!-- FxA pre-boarding header -->
```

Comment:
I think we should type out Firefox Account: I don't know that localizers know what FxA is.